### PR TITLE
Yeti-fy Forms

### LINKED ISSUE #102: Login and Register Forms Out of Date
These forms were never updated to use the new bootswatch theme.

### PR DESCRIPTION
**Pull request by rickbau5**
Resolves #102. Update the following forms to use the Bootswatch css stuff, currently Yeti. Will work for future bootswatch themes as well.
- Register
- Login
- Request password reset
- Reset password

Removed a lot of unused CSS from main.css that related to these pages.

@uidaho/lambda ready for review

### CODE REVIEW

**Comment by rickbau5** [comment]
Cleaned up commits

**Comment by brec9824** [comment]
@rickbau5 Reviewing

File: resources/views/auth/passwords/reset.blade.php [hunk 1]
```diff
@@ -1,75 +1,69 @@
...
+                    <div class="col-lg-10">
+                        <input class="form-control" type="email" name="email" id="email" value="{{ $email or old('email') }}">
+                        <!--@if ($errors->has('email'))
+                            <label class="control-label">{{ $errors->first('email') }}</label>
+                            </div>
+                        @endif -->
+                    </div>
+                </div>
+                @if($errors->has('password') or $errors->has('password_confirmation'))
+                    <div class="has-error">
```
Comment (brec9824) [inline resources/views/auth/passwords/reset.blade.php:33]: Can you line this error message up on the left like the other error messages.
error1

Comment (rickbau5) [inline resources/views/auth/passwords/reset.blade.php:33]: @brec9824 fixed!

**Comment by brec9824** [comment]
Looks good merging

**Comment by rickbau5** [comment]
:tada:

**Pull request merged**